djangocms_picture/models.py

```diff
@@ -112,12 +113,14 @@ class AbstractPicture(CMSPlugin):
         verbose_name=_('Alignment'),
         choices=PICTURE_ALIGNMENT,
         blank=True,
+        null=True,
```

Comment:
Here we should not set it to Null (CharField)

Reply:
Still curious what the issue with this is? I've seen no errors or testing... thanks!

Reply:
@carderm this adaption will require a simultaneous release with djangocms-bootstrap4 otherwise migrations will be missing from the link plugin there.